src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3531,6 +3531,11 @@ public override SyntaxNode ExpressionStatement(SyntaxNode expression)
             return SyntaxFactory.ExpressionStatement((ExpressionSyntax)expression);
         }
 
+        public override SyntaxNode ConditionalAccessExpression(SyntaxNode expression, SyntaxNode whenNotNull)
+        {
+            return SyntaxFactory.ConditionalAccessExpression(Parenthesize(expression), SyntaxFactory.MemberBindingExpression((SimpleNameSyntax)whenNotNull));
```

Comment:
1. Please wrap very long lines :)
2. No... this would not be ok.  THe problem here is that it assumes the "whenNotNull" will be a simple name.  Really, whenNotNull should be a MemberBindingExpression itself.  If you need to create a MemberBindingExpression from a simple name, then there should be a factory method for that.